src/main/java/lotto/domain/WinningNumber.java

```diff
@@ -0,0 +1,36 @@
+package lotto.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class WinningNumber {
+	public final String DELIMITER = ",";
```

Comment:
```suggestion
	public static final String DELIMITER = ",";
```

상수 컨벤션을 맞춰보시면 좋을 것 같습니다.